### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/tutorial/static-files.md`

### PR DESCRIPTION
**Pull request by wdh99**
Here also a same PR [#3828](https://github.com/tiangolo/fastapi/pull/3828), but it was two years ago, and it seems that the translator has not been to GitHub for a long time.

### CODE REVIEW

**Comment by ChoyeonChern** [comment]
很好！Well done！

File: docs/zh/docs/tutorial/static-files.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# 静态文件
+
+您可以使用 `StaticFiles`从目录中自动提供静态文件。
```
Comment (Lenclove) [inline docs/zh/docs/tutorial/static-files.md:3]: 您可以使用 `StaticFiles`目录来自动提供静态文件。

**Review by Lenclove — COMMENTED** [review]
I think it’s better for this sentence. What do you think about this?

File: docs/zh/docs/tutorial/static-files.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wdh99) [reply docs/zh/docs/tutorial/static-files.md:3]: StaticFiles不是一个目录，它是一个类，目录是这里的static`StaticFiles(directory="static")`

[the remark this answers is not in the document]
Comment (Lenclove) [reply docs/zh/docs/tutorial/static-files.md:3]: 好的，是我理解错了

**Review by Lenclove — APPROVED** [review]
approved

**Comment by wdh99** [comment]
> 很好！Well done！

兄弟, 像Lenclove那样approved一下

**Review by ChoyeonChern — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @wdh99! 🚀 

And thanks for the reviews @Lenclove and @ChoyeonChern 👏

**Pull request merged**